azure-spring-boot/pom.xml

```diff
@@ -24,12 +24,11 @@
             <artifactId>spring-boot-autoconfigure</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.hibernate.validator</groupId>
-            <artifactId>hibernate-validator</artifactId>
-            <optional>true</optional>
-        </dependency>
-
+		<dependency>
+		    <groupId>javax.validation</groupId>
+		    <artifactId>validation-api</artifactId>
+		</dependency>
```

Comment:
Hi @rajadilipkolli, thank you for sending PR to fix deprecated dependencies. Could you change the indention to align with other dependencies?

Reply:
@sophiaso , made changes to indention

Reply:
@sophiaso , made changes to indention

Reply:
@rajadilipkolli  Thank you very much for the quick response.